iso.c

```diff
@@ -302,7 +302,7 @@ iso_connect(char *server, char *username, char *domain, char *password,
 				}
 
 				logger(Core, Notice, "Retrying with plain RDP.");
-				g_negotiate_rdp_protocol = False;
+				neg_proto = PROTOCOL_RDP;
```

Comment:
I think that you should not change this line in this PR.

Reply:
Retrying with plain RDP so the neg_proto = PROTOCOL_RDP,  g_negotiate_rdp_protocol is unnecessary